components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -151,6 +152,9 @@ public static ServiceProvider build(OMElement serviceProviderOM) {
             } else if ("PermissionAndRoleConfig".equals(elementName)) {
```

Comment:
Let's use constants.

Reply:
Let's keep this as it is to maintain the consistency with the other places for now. Will fix the remaining with another PR